ftw/workspace/latex/templates/files_listing.pt

```diff
@@ -5,42 +5,27 @@
     tal:omit-tag="python: True"
```

Comment:
Usually we use Mako-Templates for LaTeX, not page templates. Is there a reason for still using page template other than convenience?

Example: https://github.com/4teamwork/ftw.meeting/blob/master/ftw/meeting/latex/templates/tasklisting.tex
(But do not use `supertabular` 😜 )